_plugins/author.rb

```diff
@@ -61,10 +61,11 @@ def with_pic(input)
       name = input[0]
       info = input[1]
       image = File.join 'assets', 'images', 'team', "#{name}.jpg"
+      @baseurl = Jekyll.sites[0].config['baseurl']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.